counter_block.py

```diff
@@ -143,17 +143,13 @@ def _calculate_next(self, now):
 
         return date
 
-    def stop(self):
-        self._store()
-        self._backup()
-
     def process_signals(self, signals):
         signals_to_notify = []
         self.for_each_group(self.process_group, signals,
                             kwargs={"to_notify": signals_to_notify})
 
         self.notify_signals(signals_to_notify)
-        self._store()
+        self._save()
```

Comment:
Good thinking. The fact that we persist on an interval means we don't need this.